lib/list/getCSVData.js

```diff
@@ -0,0 +1,110 @@
+var _ = require('lodash');
+var listToArray = require('list-to-array');
+
+/**
+ * Applies option field transforms to get the CSV value for a field
+ */
+
+function transformFieldValue (field, item, options) {
```

Comment:
I've been wondering how we could unit test this whole block of work. It has really predictable inputs and outputs. Let me know if you have any ideas on how to structure it?